examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/demo_io.c

```diff
@@ -0,0 +1,358 @@
+#include <assert.h>
```

Comment:
Nope, I didn't 🙃  I'll implement a custom error handler to loop forever on error  and get rid of the `assert()`s